### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [theroyakash/theroyakash-algorithms-comments#3] [Announcements]
# index

Materials for FAANG and top startup's coding interview preparation and running notes on that.

https://algorithms.theroyakash.com/

**Comment by theroyakash** [comment]
# Welcome
Welcome to the website, look around, find something to read or search a problem. Please comment down below your thoughts on this.

You are super welcome to contribute to this massive repository by filling out the form [here](https://docs.google.com/forms/d/e/1FAIpQLSfJVLsT_DOOnQDxRaOpi4A5zbkl4Plfe3OOnfIkGYiwPL3t0Q/viewform).

We need funding for this website to be maintained and updated continuously. If you are financially well off you can invest into us by sponsoring this project from [here](https://github.com/sponsors/theroyakash).
